src/Sylius/Behat/Resources/config/suites/ui/checkout/checkout.yml

```diff
@@ -11,13 +11,15 @@ default:
                 - sylius.behat.context.transform.lexical
                 - sylius.behat.context.transform.payment_method
                 - sylius.behat.context.transform.product
-                - sylius.behat.context.transform.shipping_method
                 - sylius.behat.context.transform.zone
+                - sylius.behat.context.transform.shared_storage
+                - sylius.behat.context.transform.shipping_method
```

Comment:
Sure we should! However I think we both agree this is something should be done i separate PR ;)